RELEASE.md

```diff
@@ -23,16 +23,68 @@ This page contains the release notes for AMD ROCm Software.
 
 #### OS and GPU support changes
 
-* AMD Instinct™ MI300A and MI300X support has been added
+* AMD Instinct™ MI300A and MI300X Accelerator support is enabled for supported OSes excluding RHEL9.
 * Added support for the following operating systems:
   * RHEL 9.3
   * RHEL 8.9
 
+#### New ROCm meta package
+
+A new ROCm meta packages has been added for easy installation of all ROCm core packages, tools, and libraries.
+For example, the following command will install the full ROCm package: `apt-get install rocm` (Ubuntu), or `yum install rocm` (RHEL).
+
+#### Filesystem Hierarchy Standard
+
+ROCm 6.0 fully adopts the Filesystem Hierarchy Standard (FHS) reorganization goals.
+The backward compatibility support for old file locations is removed.
+
+#### Compiler location change
+
+- The installation path of LLVM changes from `/opt/rocm-<rel>/llvm` to `/opt/rocm-<rel>/lib/llvm`.
+  For backward compatibility, a symbolic link is provided to the old location and will be removed in ROCm 7.0 or later.
```

Comment:
Recommended

For backward compatibility, a symbolic link is provided to the old location and will be removed in a future release.

Reply:
@Rmalavally my last change removed saad's request for you to, instead of making comments, make suggestions. We can directly commit suggestions, which is much faster/more efficient. (see Sam's suggestion just below)

Reply:
Noted.

Reply:
@LisaDelaney since you are making the editorial changes, I didn't want to override your edits. Hence, recommending the changes.